build_tools/docker/manage_images.py

```diff
@@ -60,6 +60,7 @@
     'cmake-python-nvidia': ['cmake-python-vulkan'],
     'cmake-riscv': ['cmake', 'util'],
     'cmake-bazel-frontends': ['cmake-python', 'bazel'],
+    'cmake-bazel-frontends-android': ['cmake-bazel-frontends', 'util'],
```

Comment:
Deps need to be updated

```suggestion
    'cmake-bazel-frontends-android': ['cmake-bazel-frontends', 'cmake-android'],
```

Reply:
Ah, right. Forgot about that. Done.